docs/modules/maintain-cluster/pages/enterprise-rest-api.adoc

```diff
@@ -474,3 +474,104 @@ curl -X 'DELETE' \
+}
+----
+If you look carefully in the message, it says that "there is already a conflicting configuration" with the same map name.
+
+As you can see, you can not try adding a map config with the same map name but you can send an update request with the same configuration and map name, and you will observe that this update will be ignored since there is no change. Let's try the request again:
+[source,shell]
+----
+curl -X 'POST' \
+  'http://localhost:8443/hazelcast/rest/api/v1/config/update' \
+  -H 'Content-Type: text/plain' \
+  -d '
+        <hazelcast xmlns="http://www.hazelcast.com/schema/config">
+                       <map name="my-map">
+                           <in-memory-format>BINARY</in-memory-format>
+                           <statistics-enabled>true</statistics-enabled>
+                           <backup-count>2</backup-count>
+                       </map>
+        </hazelcast>
+        '
+----
+We receive a response code OK (200) and the following body:
+[source,json]
+----
+{
+  "addedConfigs": [],
+  "ignoredConfigs": [
+    {
+      "sectionName": "map",
+      "configName": "my-map"
+    }
+  ]
+}
+----
+As you see from the response, no new configs added and the provided config is ignored.
+
+Whether a dynamic configuration can be applied or not depends on the request. For some updates, it is allowed to change configuration parameters of an existing configuration while for some as above you are not allowed to change it.
```

Comment:
Can we provide more detail on when/if configuration changes are allowed? Is there certain parameters that can be changed or some logic to it? Otherwise it's just trial and error for user.

Reply:
@oliverhowell I refer to the main section in ref manual about the dynamic configuration feature. I also added the sentence like this: 
`Please see the xref:configuration:dynamic-config.adoc[Dynamic Configuration for Members] section on details about which configuration parameters can be changed dynamically and which can not be changed.
`

We are not really trying to explain the dynamic configuration feature itself here but how you can use it using the REST API. Hence, if there are missing information on dynamic configuration options, then it should be updated on the referred feature page. It is not in this document scope but surely we can improve that side as well. I tried to capture some common cases and what the user may see as the response but I do not aim to catch all the details of dynamic config in this REST API section.

Reply:
Ah ok, makes sense. Maybe as a suggestion then we could add the 'for more info, see Dynamic Config' note to the end of this paragraph, you know the one we already use higher up the page?

Reply:
sure. changed as suggested at 624c767dd8680a5d8588518814ded2609bab7b11